packages/ketcher-core/src/application/editor/actions/closelyFusing.ts

```diff
@@ -94,7 +99,41 @@ function closestToMerge(struct, closestMap) {
     }
   })
 
+  mergeMap = omitAtomsAndBondsOfSaltsAndSolventsFromMergeMap(mergeMap, sgroups)
+
   if (mergeMap.atoms.size === 0 && mergeMap.bonds.size === 0) return null
 
   return mergeMap
 }
+
+function omitAtomsAndBondsOfSaltsAndSolventsFromMergeMap(
```

Comment:
In general, LGTM. 
But the only thing I want to clarify – can't `mergeMap` include 'salt or solvents'?
From this function, I see it doesn't add salt or solvent to a map, but can't they be added before this function gets called? In lines 85-88, there new Map created from atoms and bonds of closestItem:
```
let mergeMap: MergeMap = {
    atoms: new Map(closestMap.atoms),
    bonds: new Map(closestMap.bonds)
  }
```

Reply:
Really good suggestion. Have changed to call this function a bit earlier. Thank you!